cmake-proxies/CMakeLists.txt

```diff
@@ -160,7 +160,6 @@ set_conan_vars_to_parent()
 #
 #       directory          option      symbol      req   chk   version
 addlib( libsndfile         sndfile     SNDFILE     YES   YES   "sndfile >= 1.0.28" )
-addlib( libsoxr            soxr        SOXR        YES   YES   "soxr >= 0.1.1" )
```

Comment:
I would prefer not to have this change, as it will conflict with the devendoring project.

Reply:
I agree, we should keep certain approach for new build "Conan" conception.